src/components/resumes/Details.jsx

```diff
@@ -224,6 +227,15 @@ function Details() {
               variant="standard"
             />
           </div>
+          <div>
+            {data.skills.map((skill) => (
```

Comment:
We need a title on the skill section. You can edit the height of the Details Card if needed